umbraco/Umbraco-CMS
# V15: Handle empty permissions

# Notes
- Handle empty granular permissions for user groups.

# How to test
- Create a document type with allowed as root set to true
- Create some content with that doc type
- Go to User -> User groups, select a user group
- under granular permission, select the content, but don't select any permissions
- Save and refresh the page
- The granular permissions should persist
